.gitlab-ci.yml

```diff
@@ -179,9 +179,7 @@ test:kover:
 
 publish:release:
   tags: [ "runner:main" ]
-  only:
-    - tags
-    - develop
+  when: manual
```

Comment:
Let's be careful not to propagate this into develop, as this would break future releases.
Also you need to do the same change for all artifacts (at least all the ones used by shopist, oncall app, RN and Flutter)

Reply:
Yes, that's the reason this is just a draft.